client/src/components/DecisionButtons.vue

```diff
@@ -117,10 +117,24 @@ export default {
       this.warnDecision = false;
     },
   },
+  mounted() {
+    if (!this.currentViewData.currentAutoEvaluation) {
+      this.pollInterval = setInterval(this.pollForEvaluation, 1000 * 10);
+    }
+  },
+  beforeUnmount() {
+    clearInterval(this.pollInterval);
+  },
   methods: {
     ...mapMutations([
       'updateExperiment',
+      'setFrameEvaluation',
     ]),
+    async pollForEvaluation() {
+      const frameData = await djangoRest.frame(this.currentViewData.currentFrame.id);
+      this.setFrameEvaluation(frameData.frame_evaluation);
+      clearInterval(this.pollInterval);
```

Comment:
This looks like it will stop the interval loop immediately after the first evaluation. Are you relying on an exception being thrown somewhere to stop that from happening? If so an explicit try/catch would be nice

Reply:
The intention is that there's only ever one NN evaluation that we need to fetch, so we can stop polling once we get it.

Reply:
After getting clarification, I understand the point made here better and I added an if statement in [8fe070a](https://github.com/OpenImaging/miqa/pull/379/commits/8fe070a46eda4753de737552ccc07321713049e3) to address this